### PR TITLE
add definition files, tools and update dev-env

### PR DESCRIPTION
**Pull request by sculove**
### Issues
#40 #43 #42

### task
- fix image size in README.md
- update common-demo files for demo site.
- update dependency modules
- add sourceMap env
- add definition files and tools.

### CODE REVIEW

File: src/AxisManager.ts [hunk 1]
```diff
@@ -1,9 +1,16 @@
 import Coordinate from "./Coordinate";
+import { AxesOption } from "./Axes";
 
 export interface Axis {
   [key: string]: number
 };
 
+export interface AxisOption {
```
Comment (jongmoon) [inline src/AxisManager.ts]: It this re-definition?

File: declaration/AxisManager.d.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { AxesOption } from "./Axes";
+export interface Axis {
+    [key: string]: number;
+}
+export interface AxisOption {
```
Comment (jongmoon) [inline declaration/AxisManager.d.ts]: redefinition?

Comment (sculove) [inline declaration/AxisManager.d.ts]: `declaration` directory is only definition file.
`src` directory is only source.

File: src/AxisManager.ts (hunk 1 above)
Comment (jongmoon) [inline src/AxisManager.ts]: Sorry , my eyes had a bug.

I think `AxesOption` and `AxisOption` are the same.

File: declaration/AxisManager.d.ts (hunk 2 above)
Comment (jongmoon) [inline declaration/AxisManager.d.ts]: @sculove I thought AxesOption and AxisOption are the same. It's my mistake. Sorry.

**Comment by sculove** [comment]
CI Build successed. but, Github does not recognize it.


So, I will merge.

**Pull request merged**